db/migrate/20211020062413_create_question_answer_votes.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class CreateQuestionAnswerVotes < ActiveRecord::Migration[6.1]
+  def up
+    create_table :question_answer_votes do |t|
+      t.integer :post_id, null: false
+      t.integer :user_id, null: false
+      t.datetime :created_at, null: false
+    end
+
+    add_index :question_answer_votes, [:post_id, :user_id], unique: true
```

Comment:
We've decided to not support multiple votes per user. Too complicated for V0 for us.